src/views/text_view.rs

```diff
@@ -146,6 +171,18 @@ impl TextContentInner {
             Some(ref last) => last.x.accept(size.x) && last.y.accept(size.y),
         }
     }
+
+    fn get_value(&self) -> Arc<StyledString> {
```

Comment:
Let's return a `&mut Arc` instead, and let users clone it if needed.
This will enable the `Arc::make_mut` in `append()` above.